src/IsoscrollerNode.js

```diff
@@ -95,7 +95,7 @@
       this.textCanvas.height = 128;
 
       this.textCtx = this.textCanvas.getContext('2d');
-      this.textCtx.font = '40pt vcr';
+      this.textCtx.fontFamily = '40pt vcr';
```

Comment:
This can't be right? This will make font-family first try to load the font called '40pt' followed by 'vcr'